### PR TITLE
[EVPN] New `resource/opentelekomcloud_enterprise_vpn_customer_gateway_v5`

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
## PR Checklist

* [x] Refers to: #2640 
* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.
* [x] Release notes added.

## Acceptance Steps Performed

```
=== RUN   TestAccCustomerGateway_basic
=== PAUSE TestAccCustomerGateway_basic
=== CONT  TestAccCustomerGateway_basic
--- PASS: TestAccCustomerGateway_basic (48.87s)
PASS

Process finished with the exit code 0
```

### CODE REVIEW

File: docs/resources/vpn_customer_gateway_v5.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+* `created_at` - The create time.
+
+* `updated_at` - The update time.
+
+* `route_mode` - Specifies the routing mode.
+
+* `ip` - Specifies the IP address of the customer gateway.
+
+* `region` - Specifies the region in which to create the resource.
```
Comment (artem-lifshits) [inline docs/resources/vpn_customer_gateway_v5.md]: In which resource is created

File: opentelekomcloud/services/vpn/resource_opentelekomcloud_enterprise_vpn_customer_gateway_v5.go [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+			"tags": common.TagsSchema(),
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The create time.`,
+			},
+			"updated_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The update time.`,
```
Comment (artem-lifshits) [inline opentelekomcloud/services/vpn/resource_opentelekomcloud_enterprise_vpn_customer_gateway_v5.go]: Please remove desc

**Review by artem-lifshits — CHANGES_REQUESTED** [review]

File: opentelekomcloud/services/vpn/resource_opentelekomcloud_enterprise_vpn_customer_gateway_v5.go (hunk 2 above)
Comment (anton-sidelnikov) [reply opentelekomcloud/services/vpn/resource_opentelekomcloud_enterprise_vpn_customer_gateway_v5.go]: done

File: docs/resources/vpn_customer_gateway_v5.md (hunk 1 above)
Comment (anton-sidelnikov) [reply docs/resources/vpn_customer_gateway_v5.md]: done

**Review by artem-lifshits — APPROVED** [review]

**Pull request merged**